libs/langgraph/langgraph/pregel/loop.py

```diff
@@ -747,6 +789,7 @@ def _put_checkpoint(self, metadata: CheckpointMetadata) -> None:
                 **self.checkpoint_config,
                 CONF: {
                     **self.checkpoint_config[CONF],
+                    CONFIG_KEY_CHECKPOINT_ID: self.checkpoint_id_prev,
```

Comment:
Wasn't obvious to me initially that we could guarantee that self.checkpoint_id_prev had been set by the point this branch was run